experimenter/tests/integration/nimbus/ios/test_ios_integration.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import os
+import subprocess
+import time
+from pathlib import Path
+
+import pytest
+
+here = Path(__file__)
+
+
+@pytest.fixture
+def experiment_slug():
+    return "firefox-ios-integration-test"
+
+
+@pytest.fixture(name="device_control", scope="module", autouse=True)
+def fixture_device_control(xcrun):
+    ...
+
+
+@pytest.fixture(name="experiment_url", scope="module")
+def fixture_experiment_url(request, variables):
+    ...
+
+
+@pytest.fixture(name="experiment_data")
+def fixture_experiment_data(experiment_url, request):
+    ...
```

Comment:
What are these for?

Reply:
I need to override these fixtures, they are a part of the klaatu workflow but are not needed here.

Reply:
Oh I see, interesting.  Maybe add a comment to each that says that?